src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,5 +12,5 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-
+    public static final Prefix PREFIX_PHOTO = new Prefix("ap/");
```

Comment:
How about using p/

Reply:
I was thinking p/ clashes with phone prefix, but it could be done also